Dockerfile

```diff
@@ -55,29 +55,30 @@ RUN rm -rf valhalla
 
 ####################################################################
 # copy the important stuff from the build stage to the runner image
-FROM ubuntu:24.04 as runner
-MAINTAINER Kevin Kreiser <kevinkreiser@gmail.com>
+FROM ubuntu:24.04 AS runner
+LABEL org.opencontainers.image.authors="Kevin Kreiser <kevinkreiser@gmail.com>"
 
 # basic paths
 ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$PATH
 ENV LD_LIBRARY_PATH=/usr/local/lib:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/lib32:/usr/lib32
 
 # github packaging niceties
-LABEL org.opencontainers.image.description = "Open Source Routing Engine for OpenStreetMap and Other Datasources"
-LABEL org.opencontainers.image.source = "https://github.com/valhalla/valhalla"
+LABEL org.opencontainers.image.description="Open Source Routing Engine for OpenStreetMap and Other Datasources"
+LABEL org.opencontainers.image.source="https://github.com/valhalla/valhalla"
+
+# we need to add back some runtime dependencies for binaries and scripts
+# install all the posix locales that we support
+RUN export DEBIAN_FRONTEND=noninteractive && apt update && \
```

Comment:
this can be moved up before copying the build artifacts from the builder so it will always be cached